code/app.js

```diff
@@ -0,0 +1,14 @@
+var acc = document.getElementsByClassName("accordion");
+var i;
+
+for (i = 0; i < acc.length; i++) {
+  acc[i].addEventListener("click", function() {
+    this.classList.toggle("active");
+    var panel = this.nextElementSibling;
+    if (panel.style.maxHeight) {
+      panel.style.maxHeight = null;
+    } else {
+      panel.style.maxHeight = panel.scrollHeight + "px";
```

Comment:
Or, using template strings (interpolation) :
```
panel.style.maxHeight = `${panel.scrollHeight}px`;
```

Reply:
Thank you! Much better.